Add saved views instructions for monitors (#10543)
DataDog/documentation: content/en/monitors/manage_monitor.md

**Note**: For attributes with a large number of values across your monitors, use the attribute search bar to find the correct value.

> ### Saved View

Reviewer: ```suggestion
### Saved view
```

Leverage saved views to quickly jump to pre-set views in order to find monitors relevant to a given context like the monitors for your team or muted for more than 60 days:

{{< img src="monitors/manage_monitor/overview.jpg" alt="Saved Views selection"  style="width:90%;" >}}

> Saved views are visibile by everyone in your organisation.

Reviewer: ```suggestion
Saved views are visible by everyone in your organization.
```

> Technically, a Saved View keeps track of:

Reviewer: ```suggestion
Technically, a saved view keeps track of:
```

- The search query

#### Default view

{{< img src="monitors/manage_monitor/default.jpg" alt="Default view"  style="width:50%;" >}}

Your existing Manage Monitor view is your default saved view. This configuration is only accessible and viewable to you and updating this configuration does not have any impact on your organization.

You can **temporarily** override your default saved view by completing any action in the UI or when opening links to the Manage Monitor page that embed a different configuration.

> At any moment, from the default view entry in the Views panel:

Reviewer: ```suggestion
From the default view entry in the Views panel:
```
